RFC Draft.md

```diff
@@ -535,6 +535,111 @@ function subscribeToCreate() {
 }
 ```
 
+## Flatten the GraphQL operations' responses
+As alluded to in the prior suggestion, we're looking to flatten the results for the GraphQL operations to make them more easily accessible instead of the current three-levels-deep nested object. Would love to get your understanding on which option you prefer.
+
+**Current Usage (v5)**
+```ts
+async function createNewTodo() {
+  const res: GraphQLResult<Todo> = await API.graphql<GraphQLQuery<Todo>>(
+    graphqlOperation(createTodo, {
+      input: { id: uuid() },
+    })
+  );
+  // todo result is nested
+  console.log(res.data.createTodo); 
+}
+
+interface GraphQLResult<T = object> {
+    data?: T; // in the above example T is Todo
+    errors?: GraphQLError[];
+    extensions?: {
+        [key: string]: any;
+    };
+}
```

Comment:
```suggestion
```

IMO we can leave the type definitions out of the MD itself and surface them in the sandbox instead, what do you think?